kpi/views/Automated_KPI_Forecasts.view.lkml

```diff
@@ -0,0 +1,54 @@
+include: "//looker-hub/kpi/views/*"
+
+view: +automated_kpi_forecasts {
+
+  dimension_group: ds {
+    label: "Submission"
+    sql: CAST(${TABLE}.ds as DATE) ;;
+    type: time
+    timeframes: [
+      date,
+      month,
+      month_name,
+      quarter,
+      year,
+    ]
+    convert_tz: no
+    datatype: date
+  }
+
+  dimension: forecast_date {
```

Comment:
"Forecast Run Date" to make consistent with the CIs?

Reply:
Fixed.